### PR TITLE
fix(rbd): enc at rest

### PR DESCRIPTION
**Pull request by ldecarvalho-doc**

### CODE REVIEW

**Review by bene2k1 — APPROVED** [review]

File: managed-databases/postgresql-and-mysql/concepts.mdx [hunk 1]
```diff
@@ -53,6 +53,16 @@ A Database Instance is a managed database service created upon a custom base ima
 
 A [snapshot](/managed-databases/postgresql-and-mysql/how-to/manage-snapshots/) is a consistent, instantaneous copy of the Block Storage volume of your Database Instance at a certain point in time. They are designed to recover your data in case of failure or accidental alterations of the data by a user. They allow you to quickly create a new Instance from a previous state of your database, regardless of the size of the volume. Their limitation is that, unlike backups, snapshots can only be stored in the same location as the original data.
 
+## Encryption at rest
+
+Encryption at rest allows you to permanently encrypt your database data.
+
+When you enable encryption at rest, you cannot disable it after database creation. All databases, all data and all snapshots will be encrypted. Backup encryption is currently not available.
+
+During the beta phase, if chose to enable encryption after database creation, you must make an enablement request via a [support ticket](https://console.scaleway.com/support).
```
Comment (RoRoJ) [inline managed-databases/postgresql-and-mysql/concepts.mdx:62]:
```suggestion
During the beta phase, if you chose to enable encryption after database creation, you must make an enablement request via a [support ticket](https://console.scaleway.com/support).
```

File: managed-databases/postgresql-and-mysql/concepts.mdx [hunk 2]
```diff
@@ -53,6 +53,16 @@ A Database Instance is a managed database service created upon a custom base ima
...
+
+Enabling encryption incurs in the copy of the database data into a new, encrypted block volume. This takes approximately 1h per 100GB of storage. A couple seconds of downtime are also expected towards the end of the copying process, when the volumes are being switched.
```
Comment (RoRoJ) [inline managed-databases/postgresql-and-mysql/concepts.mdx:64]: There is something off about the first sentence here but I can't work out what it's meant to say?

Comment (ldecarvalho-doc) [reply managed-databases/postgresql-and-mysql/concepts.mdx:64]:
```suggestion
Enabling encryption means that your database data will be copied into a new, encrypted block volume. This takes approximately 1h per 100GB of storage. A couple seconds of downtime are also expected towards the end of the copying process, when the volumes are being switched.
```

File: managed-databases/postgresql-and-mysql/concepts.mdx [hunk 3]
```diff
@@ -53,6 +53,16 @@ A Database Instance is a managed database service created upon a custom base ima
 
 A [snapshot](/managed-databases/postgresql-and-mysql/how-to/manage-snapshots/) is a consistent, instantaneous copy of the Block Storage volume of your Database Instance at a certain point in time. They are designed to recover your data in case of failure or accidental alterations of the data by a user. They allow you to quickly create a new Instance from a previous state of your database, regardless of the size of the volume. Their limitation is that, unlike backups, snapshots can only be stored in the same location as the original data.
 
+## Encryption at rest
+
+Encryption at rest allows you to permanently encrypt your database data.
+
+When you enable encryption at rest, you cannot disable it after database creation. All databases, all data and all snapshots will be encrypted. Backup encryption is currently not available.
```
Comment (abarbare) [inline managed-databases/postgresql-and-mysql/concepts.mdx:60]:
```suggestion
When you enable encryption at rest, you cannot disable it after database creation. All databases, all data (including logs) and all snapshots will be encrypted. Logical backup encryption is currently not available.
```

File: managed-databases/postgresql-and-mysql/concepts.mdx [hunk 4]
```diff
@@ -53,6 +53,16 @@ A Database Instance is a managed database service created upon a custom base ima
...
+
+During the beta phase, if you chose to enable encryption after database creation, you must make an enablement request via a [support ticket](https://console.scaleway.com/support).
```
Comment (abarbare) [inline managed-databases/postgresql-and-mysql/concepts.mdx:62]:
```suggestion
During the beta phase, you can enable encryption on an existing database instance by opening a request via a [support ticket](https://console.scaleway.com/support).
```

File: managed-databases/postgresql-and-mysql/how-to/create-a-database.mdx [hunk 5]
```diff
@@ -36,6 +36,14 @@ Compared to traditional database management, which requires customers to provide
         <Message type="important">
           You can upgrade a standalone node to High Availability after Database Instance creation, but not the opposite.
         </Message>
+    - Enable encryption at rest if you want to ensure the permanent encryption of your database data.
+        <Message type="important">
+          When you enable encryption at rest, you cannot disable it after database creation. All databases, all data and all snapshots will be encrypted. Backup encryption is currently not available.
```
Comment (abarbare) [inline managed-databases/postgresql-and-mysql/how-to/create-a-database.mdx:41]: same as before

File: managed-databases/postgresql-and-mysql/how-to/create-a-database.mdx [hunk 6]
```diff
@@ -36,6 +36,14 @@ Compared to traditional database management, which requires customers to provide
...
+          <br /><br />
+          During the beta phase, if chose to enable encryption after database creation, you must make an enablement request via a [support ticket](https://console.scaleway.com/support).
```
Comment (abarbare) [inline managed-databases/postgresql-and-mysql/how-to/create-a-database.mdx:43]: same as before

File: managed-databases/postgresql-and-mysql/how-to/create-a-database.mdx (hunk 5 above)
Comment (ldecarvalho-doc) [reply managed-databases/postgresql-and-mysql/how-to/create-a-database.mdx:41]:
```suggestion
          When you enable encryption at rest, you cannot disable it after database creation. All databases, all data (including logs) and all snapshots will be encrypted. Logical backup encryption is currently not available.
```

File: managed-databases/postgresql-and-mysql/how-to/create-a-database.mdx (hunk 6 above)
Comment (ldecarvalho-doc) [reply managed-databases/postgresql-and-mysql/how-to/create-a-database.mdx:43]:
```suggestion
          During the beta phase, you can enable encryption on an existing Database Instance by opening a request via a [support ticket](https://console.scaleway.com/support).
```

File: managed-databases/postgresql-and-mysql/concepts.mdx [hunk 7]
```diff
@@ -53,6 +53,16 @@ A Database Instance is a managed database service created upon a custom base ima
 
 A [snapshot](/managed-databases/postgresql-and-mysql/how-to/manage-snapshots/) is a consistent, instantaneous copy of the Block Storage volume of your Database Instance at a certain point in time. They are designed to recover your data in case of failure or accidental alterations of the data by a user. They allow you to quickly create a new Instance from a previous state of your database, regardless of the size of the volume. Their limitation is that, unlike backups, snapshots can only be stored in the same location as the original data.
 
+## Encryption at rest
+
+Encryption at rest allows you to permanently encrypt your database data.
+
+When you enable encryption at rest, you cannot disable it after database creation. All databases, all data (including logs) and all snapshots will be encrypted. Logical backup encryption is currently not available.
```
Comment (jcirinosclwy) [inline managed-databases/postgresql-and-mysql/concepts.mdx:60]:
```suggestion
When you enable encryption at rest, you cannot disable it after database creation. All databases, data (including logs) and snapshots will be encrypted. Logical backup encryption is currently not available.
```

File: managed-databases/postgresql-and-mysql/concepts.mdx [hunk 8]
```diff
@@ -53,6 +53,16 @@ A Database Instance is a managed database service created upon a custom base ima
...
+
+During the beta phase, you can enable encryption on an existing Database Instance by opening a request via a [support ticket](https://console.scaleway.com/support).
+
+Enabling encryption means that your database data will be copied into a new, encrypted block volume. This takes approximately 1h per 100GB of storage. When the volumes are switched, a few seconds of downtime is also expected towards the end of the copying process.
```
Comment (jcirinosclwy) [inline managed-databases/postgresql-and-mysql/concepts.mdx:64]:
```suggestion
Enabling encryption means your database data will be copied into a new, encrypted block volume. This takes approximately 1 hour per 100 GB of storage. When switching volumes, expect a few seconds of downtime towards the end of the copying process.
```

File: managed-databases/postgresql-and-mysql/how-to/create-a-database.mdx [hunk 9]
```diff
@@ -36,6 +36,14 @@ Compared to traditional database management, which requires customers to provide
         <Message type="important">
           You can upgrade a standalone node to High Availability after Database Instance creation, but not the opposite.
         </Message>
+    - Enable encryption at rest if you want to ensure the permanent encryption of your database data.
+        <Message type="important">
+          When you enable encryption at rest, you cannot disable it after database creation. All databases, all data (including logs) and all snapshots will be encrypted. Logical backup encryption is currently not available.
```
Comment (jcirinosclwy) [inline managed-databases/postgresql-and-mysql/how-to/create-a-database.mdx:41]:
```suggestion
          When you enable encryption at rest, you cannot disable it after database creation. All databases, data (including logs), and snapshots will be encrypted. Logical backup encryption is not currently available.
```

File: managed-databases/postgresql-and-mysql/how-to/create-a-database.mdx [hunk 10]
```diff
@@ -36,6 +36,14 @@ Compared to traditional database management, which requires customers to provide
...
+          <br /><br />
+          During the beta phase, you can enable encryption on an existing Database Instance by opening a request via a [support ticket](https://console.scaleway.com/support).
+          <br /><br />
+          Enabling encryption incurs in the copy of the database data into a new, encrypted block volume. This takes approximately 1h per 100GB of storage. A couple seconds of downtime are also expected towards the end of the copying process, when the volumes are being switched.
```
Comment (jcirinosclwy) [inline managed-databases/postgresql-and-mysql/how-to/create-a-database.mdx:45]:
```suggestion
          Enabling encryption means your database data will be copied into a new, encrypted block volume. This takes approximately 1 hour per 100 GB of storage. When switching volumes, expect a few seconds of downtime towards the end of the copying process.
```

**Review by jcirinosclwy — APPROVED** [review]

File: managed-databases/postgresql-and-mysql/concepts.mdx [hunk 11]
```diff
@@ -53,6 +53,17 @@ A Database Instance is a managed database service created upon a custom base ima
 
 A [snapshot](/managed-databases/postgresql-and-mysql/how-to/manage-snapshots/) is a consistent, instantaneous copy of the Block Storage volume of your Database Instance at a certain point in time. They are designed to recover your data in case of failure or accidental alterations of the data by a user. They allow you to quickly create a new Instance from a previous state of your database, regardless of the size of the volume. Their limitation is that, unlike backups, snapshots can only be stored in the same location as the original data.
 
+## Encryption at rest
+
+Encryption at rest allows you to permanently encrypt your database data. The data is encrypted at volume level using [LUKS](https://en.wikipedia.org/wiki/Linux_Unified_Key_Setup). At Scaleway the `aes-xts-plain64` is used as the default. The management of the encryption key is done by Scaleway.
```
Comment (scw-wtimmermans) [inline managed-databases/postgresql-and-mysql/concepts.mdx:58]: I would say "the 'aes-xts-plain64' mode" or maybe just 'aes-xts-plain64'

Comment (ldecarvalho-doc) [inline managed-databases/postgresql-and-mysql/concepts.mdx:58]:
```suggestion
Encryption at rest allows you to permanently encrypt your database data. The data is encrypted at volume level using [LUKS](https://en.wikipedia.org/wiki/Linux_Unified_Key_Setup). At Scaleway `aes-xts-plain64` is used as the default. The management of the encryption key is done by Scaleway.
```

**Pull request merged**